src/main/java/duke/Parser.java

```diff
@@ -51,6 +57,18 @@ private static void list(TaskList tasks) {
         Ui.list(tasks.get());
     }
 
+    /**
+     * Find task containing search string. 
+     * 
+     * @param tasks
+     * @param searchString
+     */
+    private static void find(TaskList tasks, String searchString) {
+        List<Task> listTask = tasks.get().stream().filter(task -> task.toString().contains(searchString)).collect(Collectors.toList());
```

Comment:
Looks like u flexing your 1101 skills here, but Im not sure if you need this level of sophistication